README.markdown

```diff
@@ -34,9 +34,11 @@ If you want to get DistrictBuilder up and running quickly with demo data, you ca
 $ ./scripts/configure_va_demo
 ```
 
-Otherwise, you'll need to provide your own shapefiles and config.xml file, and then run
+Otherwise, you'll need to provide your own shapefiles and config.xml file. Put your shapefile in
+a `data` directory at the project root, put your `config.xml` in `django/publicmapping/config/` 
```

Comment:
We could use the `.gitkeep` convention to make sure that this directory exists in the repo.